tools/scanner/scan.mk

```diff
@@ -32,7 +33,10 @@ endif
 
 # Use the same image tag used in `flutter_common_docker_flags`
```

Comment:
Why do we use the same image tag here?

Reply:
The image tag is defined in the `flutter_common_docker_flags`. The commands in `scan.mk` uses it to run the container. Since I don't want to change the existing commands so I just build a new image with some extra tools needed for code scanner and use the same tag.